packages/common/src/services/audius-backend/solana.ts

```diff
@@ -802,7 +806,9 @@ export const createVersionedTransaction = async (
 ) => {
   const addressLookupTableAccounts = await getLookupTableAccounts(
     audiusBackendInstance,
-    { lookupTableAddresses }
+    {
```

Comment:
?? this linting doesn't feel necessary but 🤷

Reply:
I tried to get it to reformat and it wouldn't do it. I can just undo the change.

Reply:
Fixed it